docs/in_stream.rst

```diff
@@ -1,6 +1,6 @@
 nidaqmx.task.in_stream
 ======================
 
-.. automodule:: nidaqmx._task_modules.in_stream
+.. automodule:: nidaqmx.task._in_stream
```

Comment:
This is confusing.

The file says `nidaqmx.task.in_stream`, but that module does not exist. (It never did, so this was not a new problem.)

The actual help says `nidaqmx.task._in_stream.InStream`, which is correct, but it is a private submodule.

I think we should do one of the following:
- Make `nidaqmx.task.in_stream` public to match the docs
- Use [:canonical:](https://www.sphinx-doc.org/en/master/usage/domains/python.html#directive-option-py-function-canonical) to make the docs match the public alias.

Even if you make `in_stream` public, we still need to solve this for `Task`, and `:canonical:` may help.

I can't find the `Task` class on https://nidaqmx-python--534.org.readthedocs.build/en/534/task.html

Reply:
@bkeryan I've removed `in_stream.rst` since setting `__all__ = ['Task', 'InStream']` will allow `InStream` to be available as `nidaqmx.task.InStream`. Documentation looks something like this:

![image](https://github.com/ni/nidaqmx-python/assets/55676809/eb1f8c4a-858b-42c6-aba2-f44fd1a3bc32)